src/components/EditTopicList.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <ul class="topic-list">
+    <h1>🍅</h1>
```

Comment:
Why is there a cute little tomato 🍅 here?

Reply:
🥲